common/cli.py

```diff
@@ -97,7 +97,7 @@ def errorHandler(msg):
             return False
         okay()
 
-    test = 'Check/prepair snapshot path'
+    test = 'Check/prepare snapshot path'
```

Comment:
Do we need to take care of translations here? Or is this a debug message that never gets translated?